.brigade/project.yaml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://schemas.brigade.sh/schemas-v2/project.json
+apiVersion: brigade.sh/v2
+kind: Project
+metadata:
+  id: brigade-blog
+description: Automated daily re-deploy of the blog
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/cron
+    types:
+    - daily-redeploy
```

Comment:
I've already configured our cron event source to emit this event at 6 AM UTC. Will probably change the time later, but that won't require any changes to the project definition or script.